content/influxdb/clustered/write-data/best-practices/data-lifecycle.md

```diff
@@ -0,0 +1,142 @@
+
+Our recommendation is to keep these values aligned and you should set these to a value which matches your organisation's backup and recovery strategy.
+
+For example a value of `6h` (6 hours) would be appropriate for running a lean catalog that only maintains references to recent data.
+The assumption here is no backup requirement.
+
+Use the following scenarios as a guide for different use cases:
+
+{{% expand "Leading edge data, no backups" %}}
+
+When only the most recent data is important and backups are not required, you can utilise a very low cutoff point for the garbage collector.
+
+```yaml
+apiVersion: kubecfg.dev/v1alpha1
+kind: AppInstance
+metadata:
+  name: influxdb
+  namespace: influxdb
+spec:
+  package:
+    # Surrounding configuration parameters have been snipped for brevity
+    spec:
+      components:
+       garbage-collector:
+         template:
+           containers:
+             iox:
+               env:
+                 INFLUXDB_IOX_GC_OBJECTSTORE_CUTOFF: '6h',
+                 INFLUXDB_IOX_GC_PARQUETFILE_CUTOFF: '6h',
+```
+
+Using a low value here means that the garbage collector service will promptly delete files from the configured object store and remove rows from the catalog.
+Resulting in a lean catalog with minimal operational overhead.
+
+{{% /expand %}}
+
+{{% expand "Leading edge data with longer retention required" %}}
+
+In this scenario, we maintain a low cutoff point for the garbage collector service.
+
+Object versioning is used to maintain the files for a longer duration, whilst allowing the catalog to no longer maintain their references.
+
+Noncurrent objects should be expired based on the requirements of your organisation.
+
+For example, an [AWS S3 lifecycle rule](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lifecycle-mgmt.html) to expire non-current objects after 90 days will look like the following:
+
+```json
+{
+    "Rules": [
+        {
+            "ID": "my-lifecycle-rule",
+            "Filter": {
+                "Prefix": ""
+            },
+            "Status": "Enabled",
+            "NoncurrentVersionExpiration": {
+                "NoncurrentDays": 90
+            }
+        }
+    ]
+}
+```
+
+The tuning of the `garbage-collector` is shown below:
+
+```yaml
+apiVersion: kubecfg.dev/v1alpha1
+kind: AppInstance
+metadata:
+  name: influxdb
+  namespace: influxdb
+spec:
+  package:
+    # Surrounding configuration parameters have been snipped for brevity
+    spec:
+      components:
+       garbage-collector:
+         template:
+           containers:
+             iox:
+               env:
+                 INFLUXDB_IOX_GC_OBJECTSTORE_CUTOFF: '6h',
+                 INFLUXDB_IOX_GC_PARQUETFILE_CUTOFF: '6h',
+```
+{{% /expand %}}
+
+{{% expand "Long retention" %}}
+
+If your organisation requires a longer retention period for files, you may wish to increase these values past their defaults.
```

Comment:
```suggestion
{{% expand "Custom Backup Window - No S3 Versioning" %}}

If your organisation cannot make use of S3 versioning policies but still requires a backup window, you must configure the garbage collector to retain the files for as long as your backup period requires.

If possible, using S3 versioning is preferred.
```

This makes it clear why you would use this over the one above I think? And a "don't do this unless you have to" pointer to get them on the right track if they can use S3 versioning.